pkg/channel/message_receiver.go

```diff
@@ -40,14 +40,14 @@ type UnknownChannelError struct {
 }
 
 func (e *UnknownChannelError) Error() string {
-	return fmt.Sprintf("unknown channel: %v", e.c)
+	return fmt.Sprint("unknown channel:", e.c)
```

Comment:
`fmt.Sprint` doesn't inject a space between inputs
```suggestion
	return fmt.Sprint("unknown channel: ", e.c)
```

Reply:
Done